docs/CONFIG-VARS.md

```diff
@@ -187,7 +187,7 @@ Viya 4 supports 2 different types of certificate generators, Cert-manager and op
 | V4_CFG_TLS_MODE | Which TLS mode to configure | string | front-door | false | Supported values: [`full-stack`,`front-door`,`disabled.`] When deploying full-stack you must set V4_CFG_TLS_TRUSTED_CA_CERTS to trust external postgres server ca. | all |
 | V4_CFG_TLS_CERT | Path to ingress certificate file | string | | false | If specified, used instead of cert-manager issued certificates | viya |
 | V4_CFG_TLS_KEY | Path to ingress key file | string | | false | Required when V4_CFG_TLS_CERT is specified | viya |
-| V4_CFG_TLS_TRUSTED_CA_CERTS | Path to directory containing only PEM encoded trusted CA certificates files | string | | false | Required when V4_CFG_TLS_CERT is specified. Must include all the CAs in the trust chain for V4_CFG_TLS_CERT. Can be used with or without V4_CFG_TLS_CERT to specify any additionally trusted CAs  | viya |
+| V4_CFG_TLS_TRUSTED_CA_CERTS | Path to directory containing only PEM encoded trusted CA certificates files | string | | false | Required when V4_CFG_TLS_MODE is set to full-stack or front-door and the deployment target is an IAC created AWS cluster using self-signed certificates and an external postgres database. Required when V4_CFG_TLS_CERT is specified. Must include all the CAs in the trust chain for V4_CFG_TLS_CERT. Can be used with or without V4_CFG_TLS_CERT to specify any additionally trusted CAs  | viya |
```

Comment:
This is even more confusing. Simply state if you're using an external database that has TLS enabled one must add the PEM cert file to this location. This applies to Open Source K8s, any 3rd party Postgres Server. Either list exceptions here as a table or something. I believe Azure and GCP work fine with an external database but AWS does not. Same with Open Source as stated earlier.